bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+
+
+        <a href="{{ url('firefox.set-as-default.thanks')}}" class="is-not-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Set as default" data-cta-position="{{ position }}">Set as default</a>
+
+        <a href="https://blog.mozilla.org/en/firefox/" class="is-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Learn more" data-cta-position="{{ position }}">Learn more</a>
+
+        {{ download_firefox_thanks(alt_copy='Download Firefox', dom_id='protocol-nav-download-firefox', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='nav') }}
+      </div>
+    </div>
+  </header>
+  <main class="c-page-main-content">
+    <section>
+    <div class="c-browser-windows">
+      <div class="c-layered-browser">
+        {% call browser_border(class='c-browser-window-top-wrapper mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser yellow mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser dark-yellow mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser darker-yellow mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+      </div>
+
+        {% call browser_border(class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='Nothing Personal') %}
+          <div class="">
+            <h4>You’re great — probably.</h4>
+            <p>We just don’t need a lot of your data to be a <strong>fast, reliable</strong> browser that keeps up with the big guys.</p>
+          </div>
+        {% endcall %}
+
+        <div class="c-thug-life-gif-wrapper mzp-l-content mzp-t-content-md">
+          <div class="c-thug-life-gif c-detached-gif">
+            <img src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
+          </div>
+        </div>
+
+        <div class="sticky-window">
+          <div class="c-thug-life-gif c-attached-gif">
+            <img src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
```

Comment:
**suggestion:** these GIFs are 3.5mb, so suggest adding `loading="lazy"` to them?
```suggestion
            <img loading="lazy" src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
```

Reply:
**issue:** this GIF animates when prefers reduced motion is set. Might it be possible to provide a static fallback images using `<picture>`?

Reply:
I'lll ask for a static image of the gif when they send over the final gif :)!